app/build.gradle

```diff
@@ -350,6 +350,9 @@ distributions {
             into("lib") {
                 from jar
             }
+            into("lib") {
+                from("enterpriseModules")
```

Comment:
personally I don't like case sensitive file/folder names, because certain OSs have problems with them (e.g. OSX, see example) and therefore I would suggest to name it simpler `enterprise`

```
/tmp > mkdir camelCase
/tmp > mkdir camelcase
mkdir: camelcase: File exists
```